sports_club/data/pos_config.xml

```diff
@@ -1,8 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo noupdate="1">
-
-    <record model="pos.config" id="pos_config_bar" forcecreate="0">
+    <record model="pos.config" id="point_of_sale.pos_config_main">
+        <field name="start_category" eval="1"/>
+        <field name="iface_available_categ_ids" eval="[Command.link(ref('pos_category_6')), Command.link(ref('pos_category_7')), Command.link(ref('pos_category_8')), Command.link(ref('pos_category_4'))]"/>
+        <field name="iface_start_categ_id" ref="pos_category_6"/>
+    </record>
+    <record model="pos.config" id="pos_config_bar">
         <field name="name">Bar</field>
+        <field name="start_category" eval="1"/>
+        <field name="limit_categories" eval="1"/>
+        <field name="iface_available_categ_ids" eval="[Command.link(ref('pos_category_2')), Command.link(ref('pos_category_1'))]"/>
+        <field name="iface_start_categ_id" ref="pos_category_1"/>
+        <field name="floor_ids" eval="[(6, 0, [ref('restaurant_floor_1')])]"/>
     </record>
 
+    <function model="pos.config" name="write">
+        <value eval="ref('point_of_sale.pos_config_main')"/>
+        <value eval="{'active': True}"/>
+    </function>
+    <function model="pos.config" name="write">
+        <value eval="ref('pos_restaurant.pos_config_main_restaurant')"/>
+        <value eval="{'active': False}"/>
+    </function>
```

Comment:
No need for a function, this could be done on the record directly, like
```suggestion
    </function>
    <record id="pos_restaurant.pos_config_main_restaurant" model="pos.config">
        <field name="active" eval="False"/>
    </record>
```